src/main/java/com/zendesk/maxwell/MaxwellConfig.java

```diff
@@ -486,6 +519,14 @@ public void validate() {
 		} catch (MaxwellInvalidFilterException e) {
 			usage("Invalid filter options: " + e.getLocalizedMessage());
 		}
+
+		if ( this.metricsDatadogType.contains("http") && StringUtils.isEmpty(this.metricsDatadogAPIKey) ) {
+			usageForOptions("please specify metrics_datadog_apikey when metrics_datadog_type = http");
+		}
+
+		if ( this.metricsDatadogType.contains("udp") && StringUtils.isEmpty(this.metricsDatadogHost) ) {
+			usageForOptions("Please specify metrics_datadog_host when metrics_datadog_type = udp" );
```

Comment:
could just default to localhost, pretty common config to run dd-agent on all yr hosts.